lib/services/database.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:brew_crew/models/brew.dart';
+import 'package:brew_crew/models/user.dart';
+
+class DatabaseService {
+  final uid;
```

Comment:
bez sensu. serwisy powinny przyjmować conajwyżej inne serwisy (dependency injection).
możesz to rozwiązać w ten sposób, że każda metoda będzie przyjmować uid usera lub DatabaseService może przyjmować instancję AuthService i za każdym razem z niej pobierać uid aktualnego usera